cmake/openocd/stm32_f4.cfg

```diff
@@ -0,0 +1,13 @@
+# This is for all ST NUCLEO with any STM32F4. Known boards at the moment:
```

Comment:
If we are to include the OpenOCD config, I would also maybe include the .svd file for the MCU we are using. It's an ARM standardized file that describes the register map and allow debugging inside registers. This file can then be loaded in the IDE when debugging (there is a way of using it in VSCode, CLion and from the command line).

https://github.com/posborne/cmsis-svd/tree/master/data/STMicro
https://wiki.st.com/stm32mpu/wiki/CMSIS-SVD_environment_and_scripts